render-test/render_test.cpp

```diff
@@ -125,6 +125,9 @@ int runRenderTests(int argc, char** argv) {
                 color = "red";
                 stats.erroredTests++;
                 printf(ANSI_COLOR_RED "* errored %s" ANSI_COLOR_RESET "\n", id.c_str());
+                if (!metadata.errorMessage.empty()) {
```

Comment:
Why there is 'errored' branch without `error` message set? It would be better to fix runner and set appropriate error messages in case of error.